bootstrap.php

```diff
@@ -423,6 +396,12 @@ private function validateRawData($raw_data)
   }
 }
 
+if (getenv('INTERCOM_PLUGIN_TEST') == '1' && !function_exists('apply_filters')) {
```

Comment:
What is `INTERCOM_PLUGIN_TEST`?

Reply:
It's set in `.travis.yml` — I'm guessing it's a variable to check if the plugin is currently in test mode?

Reply:
Yup